samples/drivers/watchdog/src/main.c

```diff
@@ -133,10 +148,14 @@ void main(void)
 
 	/* Feeding watchdog. */
 	printk("Feeding watchdog %d times\n", WDT_FEED_TRIES);
+#if WDT_MIN_WINDOW != 0
+	/* Wait open window. */
+	k_sleep(K_MSEC(WDG_FEED_INTERVAL));
+#endif
```

Comment:
You are doing more than simple add overlay files. I think you should explain better at commit message why you are changing the example or split the changes.

Reply:
I got it. Splitting this commit.